src/AppBundle/DataFixtures/ORM/LoadFixtures.php

```diff
@@ -91,11 +96,114 @@ private function loadPosts(ObjectManager $manager)
             }
 
             $manager->persist($post);
+
+            $this->addReference('post-' . $i, $post);
```

Comment:
using the reference system while being in the same fixture class is overkill: you don't clear the entity manager, so you could simply keep the same objects. Return an array of posts from this method, and pass them to the other method